zstd/enc_best.go

```diff
@@ -343,7 +344,7 @@ encodeLoop:
 		if best.rep > 0 {
 			var seq seq
 			seq.matchLen = uint32(best.length - zstdMinMatch)
-			if debugAsserts && s <= nextEmit {
+			if debugAsserts && s < nextEmit {
 				panic("s <= nextEmit")
```

Comment:
The panic message, probably should also be updated?

Reply:
Agree